src/compiler.ts

```diff
@@ -9518,10 +9518,9 @@ export function defaulType(nativeType: NativeType, module: Module): ExpressionRe
     case NativeType.I64: return module.i64(0);
     case NativeType.F32: return module.f32(0);
     case NativeType.F64: return module.f64(0);
-    case NativeType.V128: return module.v128(new Uint8Array);
+    case NativeType.V128: return module.v128(new Uint8Array());
     case NativeType.I32:
     case NativeType.Anyref:
-    case NativeType.Exnref:
```

Comment:
It seems this wrong remove. Your tests fail in other reason. Try to syc with master. update binaryen and rebuild

Reply:
Well that will be caught by the default case anyway, but I thought it was complaining about that line when in fact it was two above when I didn't close the parenthesis of the constructor for unit8array.  I'm about to push another commit soon.  I'm adding abstract methods.

Reply:
Yes, it covered by default case but I think has all cases explicitly is just good practice

Reply:
Well actually it looks like I now longer need this anyway, so I'm going to get rid of it.